polaris/polaris/utils.py

```diff
@@ -420,13 +419,25 @@ def create_transaction_envelope(transaction, source_account) -> TransactionEnvel
         source_account=source_account,
         network_passphrase=settings.STELLAR_NETWORK_PASSPHRASE,
         base_fee=base_fee,
-    ).append_payment_op(
-        destination=transaction.stellar_account,
-        asset_code=transaction.asset.code,
-        asset_issuer=transaction.asset.issuer,
-        amount=str(payment_amount),
-        source=transaction.asset.distribution_account,
     )
+    if transaction.status == Transaction.STATUS.pending_trust:
```

Comment:
So right now, `transaction.status` is only `pending_trust` when `destination_exists=False` and `get_or_create_stellar_destination_account()` returns `pending_trust`. `destination_exists=True` might no longer used since we deleted `check_trustlines`, in which case we should remove it and always check for the account's existence.

Reply:
I think I like this path much more. 
Let me make some changes to see if it works as I intend it to.